default.nix

```diff
@@ -18,6 +18,7 @@ in pkgs.poetry2nix.mkPoetryApplication {
 
   propagatedBuildInputs = [
     pkgs.openssh
+    pkgs.rsync
```

Comment:
It's probably better to use `propagatedUserEnvPkgs` for these, that should ensure nix-env and environment.systemPackages include the binaries along with nixops.

Reply:
`propagatedBuildInputs` are wrapped by the Python builder, `rsync` shouldn't necessarily end up in `$PATH` but `nixops` needs to know where to find it.